### ISSUE TITLE
Implement date browse functionality

### ISSUE THREAD

**Issue by kaylaondracek** [mlibrary/fishrappr#137]
## Description: 
Implement date browse functionality.

See wireframes here: https://projects.invisionapp.com/share/VS73TWYC2#/screens/172996105
See Research Track ticket here: https://github.com/mlibrary/fishrappr/issues/89

## Acceptance Criteria:

- [x] a. Users should be able to navigate to browse screen via horizontal navigation on home page, search/catalog page, image viewer, help pages, etc
- [x] b. Users should be able to browse by decade, year, month, day
- [x] c. Front page for all relevant issues should appear on screen
- [x] d. User should be taken to front page for that issue when selected

**Comment by gordonleacock** [comment]
We modified the expectations for this issues - that someone can browse any day (date) as well as month, year, and decade.

I think it would be clearer if we used "Any Date" instead of "Any Day". Day references to Monday, Tuesday, etc. which we aren't tracking in our metadata. (Might in interesting through.) Let me know if I'm being way too picky.

Still a little crude (needs future work on):
Doesn't adjust dates in month for shorter months, e.g., February.
Display section refers to "search results" not "browse results"
On empty results, presents a link that, if clicked, returns to the catalog index page.
Image placeholders are displayed instead of actual page images even for "the michigan daily" publication.

**Comment by respinos** [comment]
Interesting. `Date` reads as more precise to me, and leaned toward `day` as in 

```
 %d     The day of the month as a decimal number (range 01 to 31).
              (Calculated from tm_mday.)
```

**Comment by kaylaondracek** [comment]
This ticket is done for sprint 8 - move additional acceptance criteria added after sprint start to new ticket

**Comment by gordonleacock** [comment]
Updated Acs, this is what is left:
 - [ ] When one clicks an issue and goes to the image viewer page, that page has links "Back to Results" and "Start Over" which both go to the catalog index view, not back to the browse page. Covered in issue #165.
 - [ ] Decade drop down options show "1960" instead of "1960s", etc..
 - [ ] Improve labeling or description so it is clear we are browsing **issues** not pages.

**Comment by kaylaondracek** [comment]
Closed - additional ACs moved to new ticket #168
